.github/workflows/test.yaml

```diff
@@ -39,6 +40,7 @@ jobs:
       fail-fast: false
       matrix:
         python-version:
+          - "3.10"
```

Comment:
It would be good to update the `.mergify.yaml` file to require that these new jobs pass, too.